python/refactor/utils/data/dataloader/dataloader.py

```diff
@@ -0,0 +1,169 @@
+import torch as th
+from torch.utils.data.dataloader import DataLoader
+
+class CustomizeDataLoader:
```

Comment:
the name is confusing. it seems to be used to create pytorch dataloader. it doesn't load data from files either. can we call it DataLoaderGenerator?